deploy/cloudformation/manifest-pipeline-pipeline.yml

```diff
@@ -195,6 +195,20 @@ Resources:
                 - cloudformation.amazonaws.com
             Action: 'sts:AssumeRole'
       Path: /
+      Policies:
```

Comment:
Looking at this again, seems like you could just add this as an additional statement to the `CloudFormationRolePolicy` below https://github.com/ndlib/marble-blueprints/pull/136/files#diff-7e503499d11d1b227751d6f4ede596b2L203